tests/pyccel/test_pyccel.py

```diff
@@ -659,7 +659,15 @@ def test_return_numpy_arrays(language):
     types += [bool]*2 # 2 bools for d
     pyccel_test("scripts/return_numpy_arrays.py", language=language, output_dtype=types)
 
-@pytest.mark.xfail()
+@pytest.mark.parametrize( 'language', (
+        pytest.param("fortran", marks = pytest.mark.fortran),
+        pytest.param("c", marks = [
+            pytest.mark.xfail(reason="Unravelling loops is not working yet."),
```

Comment:
Can we change this and the one above. It is not that the unravelling isn't working, it's that we don't want a hidden allocation. This is the desired behaviour. So here we should have:
```
            pytest.mark.skip(reason="Hidden memory allocation triggers pyccel error"),
```